### PR TITLE
Hide next/prev block buttons on mobile lists and search results

### PR DESCRIPTION
**Pull request by buberdds**
Closes https://github.com/oasisprotocol/explorer/pull/1221

### CODE REVIEW

File: src/app/pages/BlockDetailPage/index.tsx [hunk 1]
```diff
@@ -89,8 +89,12 @@ export const BlockDetailView: FC<{
       <dd>
         <BlockLink scope={block} height={block.round} />
         <CopyToClipboard value={block.round.toString()} />
-        <PrevBlockButton scope={block} currentRound={block.round} />
-        <NextBlockButton scope={block} currentRound={block.round} />
+        {!standalone && (
```
Comment (lukaw3d) [inline src/app/pages/BlockDetailPage/index.tsx:92]: Issue: https://5603e983.oasis-explorer.pages.dev/search?q=2%2C265%2C700
still appears on search results too

Comment (lukaw3d) [reply src/app/pages/BlockDetailPage/index.tsx:92]: (sidenote, i'm confused what standalone means.)

Comment (lukaw3d) [reply src/app/pages/BlockDetailPage/index.tsx:92]: I would expect standalone to mean single block view, not the opposite https://5603e983.oasis-explorer.pages.dev/mainnet/sapphire/block/2265700

Comment (buberdds) [reply src/app/pages/BlockDetailPage/index.tsx:92]: hard to say what it means, but I think naming does not matter now. Let's fix buttons and imo let's refactor vertical tables to rely only on css. This way we will get rid of that prop (unless it is used in more contexts that I am  not aware of)

Comment (buberdds) [reply src/app/pages/BlockDetailPage/index.tsx]: added `enableBlockNavigation` prop

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**